README.md

```diff
@@ -47,6 +49,35 @@ $ cargo test
 $ run_docker_ci_job # optional (--prune)
 ```
 
+## Langugage Spec(ish)
+
+### Statements
+
+`println` - Prints to the standard output, with a new line
+
+Example:
+
+```
+println(1+2);
+println(1);
+```
+
+### Variables
+
+Variable names:
+
+1. Variable names have to start with "_"
```

Comment:
`_` seems like an odd prefix to me, but it's up to you :)

Reply:
I agree it's an odd choice